docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1016,6 +1016,41 @@ $ helm upgrade alluxio -f config.yaml \
   alluxio-charts/alluxio
 ```
 
+{% accordion posixHelm %}
+  {% collapsible Advanced POSIX API Configuration %}
+- Alluxio fuse/client configuration:
+```properties
+properties:
+  alluxio.fuse.jnifuse.enabled: true
+  alluxio.user.metadata.cache.enabled: true
+  alluxio.user.metadata.cache.expiration.time: 2day
+  alluxio.user.metadata.cache.max.size: "1000000"
+  alluxio.user.direct.memory.io.enabled: true
+  alluxio.fuse.logging.threshold: 1000ms
+```
+- Alluxio fuse java opts
+```properties
+fuse:
+  jvmOptions: " -Xmx16G -Xms16G -XX:MaxDirectMemorySize=32g"
+```
+- Alluxio fuse mount options
+```properties
+fuse:
+  args:
+    - fuse
+    - --fuse-opts=kernel_cache,ro,max_read=131072,attr_timeout=7200,entry_timeout=7200
+```
+- Alluxio fuse environment variables
+```properties
+fuse:
+  env:
+    MAX_IDLE_THREADS: "64"
+```
+
+[POSIX API docs]({{ '/en/api/POSIX-API.html' | relative_url }}) provides more details about how to configure Alluxio POSIX API.
+  {% endcollapsible %}
+{% endaccordion %}
+
 {% endnavtab %}
 {% navtab kubectl %}
```

Comment:
@Xenorith Is it possible to embedd `navtab` and `accordion` layout together?

Reply:
yes you can put `accordion`s in `navtab`s. for example: https://docs.alluxio.io/ee/user/stable/en/hybrid-cloud/AWS-Advanced-Guide.html#specific-module-input-variables-and-outputs